Central-Server/API/static/ws_page.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<head>
+
+</head>
+<body>
+    DATASTREAMER WEBSOCKET STATUS OK

```

Comment:
what is this page used for?

Reply:
wsgi needs to serve content, this doubles as a way to check if the ws server is active